HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -108,36 +112,31 @@ string followUpLink
-                MemoryStream recentReferralRequestOverviewFileData,
-                MemoryStream recentReferralRequestFollowUpFileData,
-                MemoryStream historicReferralRequestFollowUpFileData
-                )
-                {
-                    var recipientList = govUkNotifyConfig.ComplianceEmailRecipients;
-                    if (String.IsNullOrEmpty(recipientList))
-                    {
-                        return;
-                    }
-                    
-                    var template = govUkNotifyConfig.ComplianceReportTemplate;
-                    Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
-                    {
-                        { "File1Link", NotificationClient.PrepareUpload(recentReferralRequestOverviewFileData.ToArray(), true)},
-                        { "File2Link", NotificationClient.PrepareUpload(recentReferralRequestFollowUpFileData.ToArray(), true)},
-                        { "File3Link", NotificationClient.PrepareUpload(historicReferralRequestFollowUpFileData.ToArray(), true)},
-                    };
-                    foreach (var emailAddress in recipientList.Split(","))
-                    {
-                        var emailModel = new GovUkNotifyEmailModel
-                        {
-                            EmailAddress = emailAddress.Trim(),
-                            TemplateId = template.Id,
-                            Personalisation = personalisation
-                        };
-                        SendEmail(emailModel);
-                    }
-                    
-                }    
+            MemoryStream recentReferralRequestOverviewFileData,
+            MemoryStream recentReferralRequestFollowUpFileData,
+            MemoryStream historicReferralRequestFollowUpFileData)
+        {
+            var recipientList = govUkNotifyConfig.ComplianceEmailRecipients;
+            var template = govUkNotifyConfig.ComplianceReportTemplate;
+            var personalisation = new Dictionary<string, dynamic>
+            {
+                { template.File1Link, NotificationClient.PrepareUpload(recentReferralRequestOverviewFileData.ToArray(), true)},
+                { template.File2Link, NotificationClient.PrepareUpload(recentReferralRequestFollowUpFileData.ToArray(), true)},
+                { template.File3Link, NotificationClient.PrepareUpload(historicReferralRequestFollowUpFileData.ToArray(), true)},
+            };
+            SendEmailToRecipients(recipientList, template.Id, personalisation);
+        }
+
+        public void SendPendingReferralReportEmail()
+        {
+            var recipientList = govUkNotifyConfig.PendingReferralEmailRecipients;
+            var template = govUkNotifyConfig.PendingReferralReportTemplate;
+            var personalisation = new Dictionary<string, dynamic>
+            {
+                { template.LinkPlaceholder, "https://www.gov.uk/apply-home-upgrade-grant" },
```

Comment:
I didn't think to ask before, but is there a reason that you are providing this as personalisation? If it's always just constant, it should just be defined in the template.

Reply:
for now just since it will stop being a constant soon & so I left a gap for the download link to go when the code to generate it comes in

Reply:
Oh I see, sorry. I was forgetting how the attachments worked. Thanks. 👍